### ISSUE TITLE
Can't replicate results

### ISSUE THREAD

**Issue by rusxqkey** [huyvnphan/PyTorch-CIFAR10#5]
Hi,

I verified the noted results of 94.07% percent of the vgg16_bn pretrained net. However, when trying to train the vgg16_bn net from scratch using the noted training parameters (minibatch 256, sgd nesterov momentum 0.9, weight decay 0.001, step learning with reduction factor 0.1 every 200 epochs, 600 epochs total) and the given normalization values, I only achieved best at epoch 332 with validation set acc 0.896300 and training set acc 1.000000. Gap of 0.103700.

How come?

![learning_rate_curve_20200311T115727853857Z](https://user-images.githubusercontent.com/62146662/76633118-831b3000-6544-11ea-977b-7aef8ac0453d.png)
![training_curve_20200311T115715797000Z](https://user-images.githubusercontent.com/62146662/76633127-857d8a00-6544-11ea-89fe-166602f213b3.png)

**Comment by huyvnphan** [comment]
Hi. It has been a while since I update the code. Let me take a look at it and get back to you in a week or two.
